api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -131,25 +125,18 @@ public void alertUserFreeTierDollarThreshold(
         buildHtml(
             FREE_TIER_DOLLAR_THRESHOLD_RESOURCE,
             freeTierDollarThresholdSubstitutionMap(user, currentUsage, remainingBalance));
-    final String subject =
-        String.format(
-            "Reminder - %s Free credit usage in All of Us Researcher Workbench",
-            formatPercentage(threshold));
-
-    final MandrillMessage msg =
-        new MandrillMessage()
-            .to(Collections.singletonList(validatedRecipient(user.getContactEmail())))
-            .html(msgHtml)
-            .subject(subject)
-            .fromEmail(workbenchConfig.mandrill.fromEmail);
 
     sendWithRetries(
-        msg, String.format("User %s passed a free tier dollar threshold", user.getUsername()));
+        user.getContactEmail(),
+        String.format(
+            "Reminder - %s Free credit usage in All of Us Researcher Workbench",
+            formatPercentage(threshold)),
+        String.format("User %s passed a free tier dollar threshold", user.getUsername()),
+        msgHtml);
   }
 
   @Override
   public void alertUserFreeTierExpiration(final DbUser user) throws MessagingException {
-    final WorkbenchConfig workbenchConfig = workbenchConfigProvider.get();
 
     final String logMsg =
         String.format("Free credits have expired for User %s", user.getUsername());
```

Comment:
nit: make a String constant for this message and use it in the description arg as well when calling sendWithRetries

Reply:
nice